node/Dockerfile.node_builder12

```
@@ -0,0 +1,31 @@
+FROM node:12-alpine
+
+LABEL maintainer="tech@flow.io"
+
+# Default to UTF-8 file.encoding
+ENV LANG C.UTF-8
+
+ENV NODE_ENV production
+ENV GOPATH /root/go
+ENV GOBIN $GOPATH/bin
+ENV PATH $PATH:$GOBIN
+
+ARG GITHUB_TOKEN
+
+WORKDIR /root
+
+RUN npm install -g forever && \
+    apk add --no-cache --update python3 py-pip ca-certificates musl-dev git go make build-base && \
+    pip install --upgrade awscli && \
+    apk -v --purge del py-pip
```

```suggestion
    pip install --upgrade awscli
```

This command `apk -v --purge del py-pip` was removed in my latest PR... It causes a python dependency `six` to be removed which the awscli depends on at runtime.